services/motion/builtin/builtin.go

```diff
@@ -145,6 +162,10 @@ func (ms *builtIn) Move(
 	constraints *servicepb.Constraints,
 	extra map[string]interface{},
 ) (bool, error) {
+	if err := ms.validateNotClosed(); err != nil {
+		return false, err
+	}
```

Comment:
These `validateNotClosed` ensures that the motion service conforms to the expectations the RDK resource system expects, which is that all model methods should return errors if the model instance is closed.